File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/StageGroupConfigModel.cs
namespace ConfigSystem
{
﻿    public class StageGroupConfigModel :ConfigModel {
        public int group_type;//章节类型
        public string scene_name;//属于场景
        public System.Collections.Generic.List<int> pre_group;//前置章节
        public string name;//章节名称
        public int open_level;//开启职级
        public System.Collections.Generic.List<int> stage_id;//拥有关卡id
        public System.Collections.Generic.List<System.Collections.Generic.List<int>> target_reward;//目标奖励
    }
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/StageGroupConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class StageGroupConfigModel extends ConfigModel {
	public int group_type;//章节类型
	public String scene_name;//属于场景
	public ArrInt pre_group;//前置章节
	public String name;//章节名称
	public int open_level;//开启职级
	public ArrInt stage_id;//拥有关卡id
	public ArrArrInt target_reward;//目标奖励
}